frontend/src/main/scala/bloop/cli/Commands.scala

```diff
@@ -34,6 +34,14 @@ object Commands {
       @Recurse cliOptions: CliOptions = CliOptions.default
   ) extends RawCommand
 
+  case class Completion(
```

Comment:
Can we call this `autocompletion` so that `completion` doesn't conflict with `compile`?

Reply:
I was gonna suggest the same, ideally there should be no conflicting tasks with `test` and `compile` to make `c<TAB> => compile` and `t<TAB>` completion work